sqrl-tools/sqrl-cli/pom.xml

```diff
@@ -57,6 +57,18 @@
       <scope>runtime</scope>
     </dependency>
     <!-- Test jars -->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>${mockito.version}</version>
```

Comment:
Remove the version field. The version field should live in the root pom and these dependency versions managed by the dependencyManagement section

Reply:
Yeah, makes sense. Thanks!